src/common/store-data-to-scan-node-result-converter.ts

```diff
@@ -105,24 +80,26 @@ function convertUnifiedStoreDataToScanNodeResults(
     return transformedResults;
 }
 
-function convertAssessmentStoreDataToScanNodeResults(
-    assessmentStoreData: AssessmentStoreData | null,
+export type ConvertAssessmentStoreDataToScanNodeResultsCallback = (
+    assessmentStoreData: AssessmentStoreData,
+    selectedTest: string,
+    cardSelectionStoreData?: CardSelectionStoreData,
```

Comment:
nit: with the refactoring changes, I think the card selection store data should not be optional.